packages/nixos/imagepuller.nix

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 Edgeless Systems GmbH
+# SPDX-License-Identifier: BUSL-1.1
+
+{
+  pkgs,
+  lib,
+  ...
+}:
+
+{
+  config = {
+    systemd.services.imagepuller = {
```

Comment:
maybe put it into kata.nix, add a config option imagePull and add this unit if value is "guest" .